pkg/npm/npm.go

```diff
@@ -14,7 +14,9 @@ import (
 )
 
 const (
-	npmBomFilename = "bom-npm.xml"
+	npmBomFilename          = "bom-npm.xml"
+	cycloneDxPackageVersion = "@cyclonedx/cyclonedx-npm@1.11.0"
+	cycloneDxSchemaVersion  = "1.4"
```

Comment:
Do we plan to watch this with renovate or why is it a constant?

Reply:
Cyclone dx has different schema versions (1.2, 13, 1.4 etc). Currently we want 1.4
So wanted to make it a constant in case in future we decide to go with next schema versions, although for other build tools in the library it is hardcoded currently to 1.4.
Should we go with hardcoded value?